### PR TITLE
Update dependencies + tweaks

### PR DESCRIPTION
**Pull request by benface**
This notably removes MIPs from the global navigation (under "Ecosystem").

### CODE REVIEW

File: packages/nextra-theme/src/index.tsx [hunk 1]
```diff
@@ -128,15 +128,12 @@ export default function NextraLayout({ children, pageOpts, pageProps }: NextraTh
...
-  if (frontMatter.socialImage) {
-    seo.openGraph = {
-      images: [{ url: frontMatter.socialImage }],
-    }
+    title: `${title ? `${title} | ` : ''}Docs | The Graph`,
+    description: frontMatter.description,
+    openGraph: {
+      title,
+      images: frontMatter.socialImage ? [{ url: frontMatter.socialImage }] : undefined,
+    },
```
Comment (benface) [inline packages/nextra-theme/src/index.tsx:136]: Simpler + makes titles consistent with `graph-website` (i.e. `{pageTitle} | {sectionTitle} | The Graph` for `<title>`, and just `pageTitle` for `og:title`)

File: website/pages/en/querying/querying-best-practices.mdx [hunk 2]
```diff
@@ -65,8 +65,6 @@ Failing to follow the above rules will end with an error from the Graph API.
 
 For a complete list of rules with code examples, please look at our GraphQL Validations guide.
 
-<br />
```
Comment (benface) [inline website/pages/en/querying/querying-best-practices.mdx:68]: The extra spacing looks weird IMO; I think we should be consistent with spacing.

File: packages/nextra-theme/package.json [hunk 3]
```diff
@@ -43,13 +43,13 @@
...
-    "@edgeandnode/organisms": "^2.9.1",
+    "@edgeandnode/gds": "^2.9.0",
+    "@edgeandnode/organisms": "^2.10.0",
     "@emotion/react": "^11.11.1",
     "@types/lodash": "^4.14.196",
-    "@types/react": "^18.2.18",
+    "@types/react": "^18.2.19",
     "@types/react-dom": "^18.2.7",
-    "next": "^13.4.12",
+    "next": "13.4.12",
```
Comment (benface) [inline packages/nextra-theme/package.json:52]: There's [an issue with Next 13.4.13](https://github.com/vercel/next.js/issues/53171), so locking the version for now.

**Review by dimaMachina — APPROVED** [review]

File: website/src/_app.tsx [hunk 4]
```diff
@@ -1,13 +1,11 @@
-import merge from 'lodash/merge'
```
[the remark this answers is not in the document]
Comment (benface) [reply website/src/_app.tsx:1]: Good catch, done in https://github.com/graphprotocol/docs/pull/462/commits/24d9a92cdee6728066be6e9ed0d026d0c51312df

**Pull request merged**